opcua/common/subscription.py

```diff
@@ -208,39 +208,58 @@ def _get_node(self, nodeid):
             node = Node(self.server, ua.NodeId(nodeid))
         return node
 
-    def _get_filter_from_event_type(self, eventtype):
-        eventtype = self._get_node(eventtype)
+    def _get_filter_from_event_type(self, eventtypes):
+        # FIXME must be a nicer way to make a list out of eventtypes when it's passed
+        # FIXME an integer (such as ua.ObjectIds.BaseEventType)
+        evtypes = []
+        if type(eventtypes) is not list:
+            evtypes.append(eventtypes)
+        else:
+            evtypes = eventtypes
+
+        evtype_node_list = []
+        for evtype in evtypes:
+            evtype_node_list.append(self._get_node(evtype))  # make sure we have a list of Node objects
+
```

Comment:
now I understand:
`
if not type(evtypes) in (list, tuple):
    evtypes = [evtypes]
evtypes = [self._get_node(i) for i in evtypes]
`